setup.py

```diff
@@ -30,6 +30,7 @@
         'pyyaml',
         'six',
         'webob',
+        'web-fragments',
```

Comment:
We'll have to be careful not to use this version of XBlock until web-fragments is on PyPI.